R/get_draws_df.R

```diff
@@ -56,7 +56,7 @@ get_draws_df <- function(ww_data,
   draws <- fit_obj$result$draws()
 
   count_draws <- draws |>
-    tidybayes::spread_draws(pred_hosp[t]) |>
+    tidybayes::spread_draws(!!str2lang("pred_hosp[t]")) |>
```

Comment:
@dylanhmorris, this change breaks `pkgdown` but not `R CMD check`. Any idea why?

Reply:
Based on the logs, it looks like it's not the one you highlighted, it's this one: https://github.com/CDCgov/ww-inference-model/pull/108/files#r1737876278

so I think it's just the missing exclamation point.